staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -181,6 +181,20 @@ type VolumeSource struct {
 	//
 	// +optional
 	Ephemeral *EphemeralVolumeSource `json:"ephemeral,omitempty" protobuf:"bytes,29,opt,name=ephemeral"`
+	// image represents a OCI object (image or artifact) pulled and mounted on the kubelet's host machine.
```

Comment:
Thinking about the guarantees we provide here, I want to suggest a sentence to include here that speaks to the consistency with the image field on containers as well as to the responsibility of the CRI implementation to resolve the details:

> image represents an OCI object (a container image or artifact) pulled and mounted on the kubelet's host machine.  

and then later on (maybe right before `The OCI object gets mounted ...`):

> The types of objects that may be mounted by this volume are defined by the container runtime implementation on a host machine and at minimum must include all valid types supported by the container image field.

Which I think is an important nuance - we want to leave it open for extension by distros, by OCI, and if someone today has a custom CRI that interprets this value it should be valid for them to do so.

Reply:
Alright, updated the docs as suggested. :+1:

Reply:
Alright, updated the docs as suggested. :+1: